### PR TITLE
Add remove functionality.

### PR DESCRIPTION
**Pull request by dzyp**
Found this library we can use for persistent caching (uses native).  Already implements LRU and localstorage using native.  Needed to add remove functionality though.

@RealKinetic/devs

### CODE REVIEW

File: src/PersistentCache.elm [hunk 1]
```diff
@@ -185,6 +185,39 @@ correctInfo key ((bits, equeue) as info) =
...
+
+    subtract maybeString =
+      case maybeString of
+        Just oldString ->
+          getSize qualifiedKey oldString
+
+        Nothing ->
+          0
+
+    combine : (Int, EQueue) -> Maybe String -> () -> ((), (Int, EQueue))
```
Comment (lyddonb) [inline src/PersistentCache.elm]: Any reason to not make this top level?

**Comment by lyddonb** [comment]
A nit but otherwise lgtm

File: src/PersistentCache.elm (hunk 1 above)
Comment (dzyp) [inline src/PersistentCache.elm]: Just pretty much confined to this specific `map2`.  The type annotation just helped me better understand what was going on, and hopefully anyone else who later has to look at this function :).  I don't really have a strong preference about where this goes though, but probably wouldn't export it either way.

**Comment by lyddonb** [comment]
+1

**Pull request merged**